e2e-tests/playwright/e2e/guest-signin-happy-path.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from '@playwright/test';
+import { UIhelper } from '../utils/UIhelper';
+import { HomePage } from '../support/pages/HomePage';
+import { Common } from '../utils/Common';
+import { templates } from '../support/testData/templates';
+
+test.describe('Guest Signing Happy path', () => {
+  let uiHelper: UIhelper;
+  let homePage: HomePage;
+  let common: Common;
+
+  test.beforeEach(async ({ page }) => {
+    uiHelper = new UIhelper(page);
+    homePage = new HomePage(page);
+    common = new Common(page);
+    await common.loginAsGuest();
+  });
+
+  test('Verify the Homepage renders with Search Bar, Quick Access and Starred Entities', async () => {
+    await uiHelper.verifyHeading('Welcome back!');
+    await uiHelper.openSidebar('Home');
+    await homePage.verifyQuickAccess('Developer Tools', 'Podman Desktop');
+  });
+
+  test.skip('Verify Catalog page renders with no components', async () => {
+    await uiHelper.openSidebar('Catalog');
+    await uiHelper.verifyHeading('My Org Catalog');
+    await uiHelper.selectMuiBox('Kind', 'Component');
+    // Check if there are no records to display.
```

Comment:
Don't we need any verification here?

Reply:
@josephca  it's skip this verification, I will need to see with @subhashkhileri what was the intention here